### DISCUSSION TITLE
How To Styling Combobox of Select Component View MantineProvider

### DISCUSSION THREAD

**Discussion by axmad386** [mantinedev/mantine#1973] [Q&A] [answered]
Inside `Select` component there is `div` with role = `combobox`. How to customize this element via mantine provider?
![image](https://user-images.githubusercontent.com/8775678/182524940-6da96cf4-4582-46a5-9668-eaa5a4eb7ce6.png)

**Comment by rtivital** [comment, marked as answer]
```tsx
<MantineProvider theme={{
  components: {
    Select: {
      styles: {
        root: {
          '& [role="combobox"]': {
            ...styles
          }
        }
      }
    }
  }
}} />
```

**Reply by axmad386** [reply]
Wow thanks, but how it works? it's like scss syntax right? @rtivital

**Reply by rtivital** [reply]
yes
